tests/common.go

```diff
@@ -7398,3 +7398,27 @@ func deleteNamespaces(namespaces []string) error {
 	}
 	return nil
 }
+
+// VerifyNilPointerDereferenceError returns true if nil pointer dereference, output of the log messages
+func VerifyNilPointerDereferenceError(n *node.Node) (bool, string) {
```

Comment:
Better return the error or log it somewhere

Reply:
this is the negative check , where it returns true if string is found in journalctl logs, in case of nil pointer error , we fail the test inside the script